ksp_plugin/ksp_plugin.vcxproj

```diff
@@ -382,35 +63,13 @@ copy /Y "$(OutDir)$(TargetName).pdb" "$(SolutionDir)$(Configuration)\GameData\Pr
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release_LLVM|Win32'">"$(OutDir)tools.exe" generate_profiles</Command>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release_LLVM|x64'">"$(OutDir)tools.exe" generate_profiles</Command>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">"$(OutDir)tools.exe" generate_profiles</Command>
-      <Command Condition="'$(Configuration)|$(Platform)'=='Release|x64'">"$(OutDir)tools.exe" generate_profiles</Command>
-      <Message Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">Generating C#/C++ files for %(FullPath)</Message>
-      <Message Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">Generating C#/C++ files for %(FullPath)</Message>
-      <Message Condition="'$(Configuration)|$(Platform)'=='Release_LLVM|Win32'">Generating C#/C++ files for %(FullPath)</Message>
-      <Message Condition="'$(Configuration)|$(Platform)'=='Release_LLVM|x64'">Generating C#/C++ files for %(FullPath)</Message>
-      <Message Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">Generating C#/C++ files for %(FullPath)</Message>
-      <Message Condition="'$(Configuration)|$(Platform)'=='Release|x64'">Generating C#/C++ files for %(FullPath)</Message>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">interface.generated.h;..\journal\player.generated.cc;..\journal\profiles.generated.h;..\journal\profiles.generated.cc;..\ksp_plugin_adapter\interface.generated.cs;%(Outputs)</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">interface.generated.h;..\journal\player.generated.cc;..\journal\profiles.generated.h;..\journal\profiles.generated.cc;..\ksp_plugin_adapter\interface.generated.cs;%(Outputs)</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Release_LLVM|Win32'">interface.generated.h;..\journal\player.generated.cc;..\journal\profiles.generated.h;..\journal\profiles.generated.cc;..\ksp_plugin_adapter\interface.generated.cs;%(Outputs)</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Release_LLVM|x64'">interface.generated.h;..\journal\player.generated.cc;..\journal\profiles.generated.h;..\journal\profiles.generated.cc;..\ksp_plugin_adapter\interface.generated.cs;%(Outputs)</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">interface.generated.h;..\journal\player.generated.cc;..\journal\profiles.generated.h;..\journal\profiles.generated.cc;..\ksp_plugin_adapter\interface.generated.cs;%(Outputs)</Outputs>
-      <Outputs Condition="'$(Configuration)|$(Platform)'=='Release|x64'">interface.generated.h;..\journal\player.generated.cc;..\journal\profiles.generated.h;..\journal\profiles.generated.cc;..\ksp_plugin_adapter\interface.generated.cs;%(Outputs)</Outputs>
+      <AdditionalInputs>$(PrincipiaGenerateProfiles)</AdditionalInputs>
```

Comment:
Should this be called `PrincipiaGenerateProfilesInputs`?

Reply:
Nah, it's the name of the actual tool (much like for `protoc` it's `$(Protoc)`)